test/autoupdate.test.js

```diff
@@ -2,94 +2,95 @@ require('./init.js');
 var assert = require('assert');
 var ds;
 
-before(function () {
+before(function() {
   ds = getDataSource();
 });
 
-describe('MS SQL server connector', function () {
-  it('should auto migrate/update tables', function (done) {
-
+describe('MS SQL server connector', function() {
+  it('should auto migrate/update tables', function(done) {
+    /* eslint-disable camelcase */
```

Comment:
Please just note: I have disabled `camelcase` checks for some variables in this file, otherwise lots of test will be broken.